app/app/room/[id].tsx

```
@@ -93,131 +95,299 @@
-              'video:',
-              mediaManager.current.getVideoInputDevices().length
-            );
-
-            // Initialize WebRTC
-            console.log('[Room] Initializing WebRTC');
+            
+            // Start enumerating devices and initializing WebRTC in parallel
+            const deviceEnumPromise = (async () => {
+              console.log('[Room] Enumerating media devices in background');
+              try {
+                const devices = await mediaManager.current!.enumerateDevices();
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)